src/components/Post/index.jsx

```diff
@@ -750,4 +729,25 @@ function Post(prop) {
   );
 }
 
+const ReadMore = ({ children, picCap = false }) => {
```

Comment:
why didn't u update the readmore component, instead removed it from import and created it here instead

Reply:
Sorry for that. I change this now